SystemSettings.php

```diff
@@ -21,6 +23,16 @@ class SystemSettings extends \Piwik\Settings\Plugin\SystemSettings
     const CUSTOM_TEMPLATES_ADMIN = 'admin';
     const CUSTOM_TEMPLATES_SUPERUSER = 'superuser';
 
+    const USER_PERMISSON_LIST = [
+        \Piwik\Access\Role\View::ID,
```

Comment:
Can we use name index ?
```suggestion
       'view' => \Piwik\Access\Role\View::ID,
```

Reply:
@AltamashShaikh I thought about doing that, but wanted to ensure the order of the array since I use the index number for comparison during the security check. If I used a name index, I would probably use a separate array to map the index name to the permission weight; that just felt like complicating things more than necessary.

Reply:
It also felt a little funny creating an array like ['view' => 'view', 'write' => 'write', ...];

Reply:
What if we make individual constants ?

Reply:
I could create constants, but I don't see much point as the constants already exist in other classes. I just thought that the array was a good way to track permission weight.

I suppose that I could use `array_search()` to return the index of the array instead of hard coding the int indexes. That might help with maintainability.

Reply:
I created a helper function that looks up the index of the string. I think that makes the code a bit easier to read at a glance :+1: